content/bn/docs/tutorials/kubernetes-basics/update/update-interactive.html

```diff
@@ -0,0 +1,33 @@
+---
+title: মিথস্ক্রিয় প্রশিক্ষণ - অ্যাপটিকে আধুনিক রূপ দেওয়া
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="bn">
+
+<body>
+
+{{< katacoda-tutorial >}}
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <div class="katacoda">
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/6" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+               <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/scale/scale-interactive/" role="button">&lt; Return to Module 5<span class=""></span></a>
+               <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">Return to Kubernetes Basics<span class=""></span></a>
```

Comment:
Kindly localize this part "Return to Kubernetes Basics"

Reply:
Done them, Thanks for the review!!